tests.py

```diff
@@ -86,3 +88,15 @@ def test_coexistence():
 
         # But does not delete the original
         assert PySide.QtGui.QStringListModel
+
+
+@with_setup(clean)
+def test_sip_api_qtpy():
+    """Qt.py with preferred binding PyQt4 should have sip version 2"""
+
+    os.environ["QT_PREFERRED_BINDING"] = "PyQt4"
```

Comment:
Nitpick, what do you think about the contextmanager, and not declaring that extra variable? The variable doesn't add any new information I think.

```python
with pyqt4():
  import Qt
  import sip
  assert sip.getapi("QString") == 2, ("PyQt4 API version should be 2, "
                                      "instead is %s" % api_version)
```